core/lyrics/lyrics.go

```diff
@@ -0,0 +1,35 @@
+package lyrics
+
+import (
+	"context"
+	"strings"
+
+	"github.com/navidrome/navidrome/conf"
+	"github.com/navidrome/navidrome/log"
+	"github.com/navidrome/navidrome/model"
+)
+
+func GetLyrics(ctx context.Context, mf *model.MediaFile) (model.LyricList, error) {
```

Comment:
I was thinking that this would be a struct, with a constructor that would allow us to inject the external.Provider (similar to artwork/artwork.go). But that's fine, I can refactor that as part of introducing the LyricsRetriever interface in the provider.

Reply:
The way I saw it was, external lyrics would just provide a string (and maybe a language) and let Navidrome parse it.

Reply:
I see it as a core service, the source of truth for lyrics. So it will have to query external providers, similar to how artwork.Artwork does it. And to be able to do it, we need to inject the external.Provider into it and it needs to hold the reference. The pattern in the codebase is to use a struct for that.